pkg/yurtadm/constants/constants.go

```diff
@@ -220,6 +223,7 @@ spec:
     - --server-addr={{.kubernetesServerAddr}}
     - --node-name=$(NODE_NAME)
     - --join-token={{.joinToken}}
```

Comment:
please don't left `join-token` in Yurthub.yaml.

Reply:
Fixed